### ISSUE TITLE
Uncaught ReferenceError: o is not defined

### ISSUE THREAD

**Issue by Dijkstra173** [steelbrain/Atom-Hack#52]
[Enter steps to reproduce below:]

1. Install Atom-Hack 
2. Create .atom-hack file
3. Open Folder

OS : Windows 8.1 / Atom 0.187.0

**Atom Version**: 0.187.0
**System**: Unknown Windows Version
**Thrown From**: [Atom-Hack](https://github.com/steelbrain/atom-hack) package, v3.4.0

### Stack Trace

Uncaught ReferenceError: o is not defined

```
At C:\Users\l\atom\packages\Atom-Hack\node_modules\node-ssh\lib\build\index.js:1

ReferenceError: o is not defined
  at new t (C:\Users\l\.atom\packages\Atom-Hack\node_modules\node-ssh\lib\build\index.js:1:330)
  at C:\Users\l\.atom\packages\Atom-Hack\lib\h.coffee:24:26
  at fs.js:337:14
  at FSReqWrap.oncomplete (fs.js:100:15)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ],
    "disabledPackages": [
      "ssh-config",
      "remote-ftp-master",
      "node-ftp-master",
      "ssh2-master"
    ]
  },
  "Atom-Hack": {}
}
```

### Installed Packages

```coffee
# User
Atom-Hack, v3.4.0

# Dev
No dev packages
```

**Comment by steelbrain** [comment]
I have a planned rewrite of `node-ssh`, it should be fixed pretty soon.

**Comment by steelbrain** [comment]
Fixed by steelbrain/node-ssh#6
